doc/all.md

```diff
@@ -0,0 +1,37 @@
+@function can-type/all all
+@parent can-type/methods 7
+@description Derive a new type that changes the types of properties using a converter function.
+
+@signature `type.all(converter, Type)`
+
+  Given a converter function, which should be one of [can-type/check], [can-type/maybe], [can-type/convert], or [can-type/maybeConvert], create a new type that inherits the same properties of `Type`, but runs the converter function on them and returns a [can-type.typeobject].
```

Comment:
Can we put
> which should be one of [can-type/check], [can-type/maybe], [can-type/convert], or [can-type/maybeConvert]

...in parentheses or in a separate sentence? I think the commas get a little confusing this way.

Also, I think
> but runs the converter function on them

is a little hard to follow.

I would phrase this something like

> Create a new type that inherits the same properties of `Type`, but with each property run through the `converter` function. The `converter` function should be one of [can-type/check], [can-type/maybe], [can-type/convert], or [can-type/maybeConvert]

Reply:
Sounds good!

Reply:
Did this.